source/extensions/common/wasm/context.h

```diff
@@ -425,6 +425,7 @@ class Context : public proxy_wasm::ContextBase,
 
   // HTTP filter state.
   bool http_request_started_ = false; // When decodeHeaders() is called the request is "started".
+  bool http_local_response_sent_ = false; // indicates if the local response is sent
```

Comment:
Is this the only case when this can happen or should we call it something more general?